challenges/1-produto-simples/rust/matheusth/.gitignore

```diff
@@ -0,0 +1 @@
+produto
```

Comment:
Por que você precisa disso?

Reply:
eu to usando essa linha pra evitar deu fazer commit do binário gerado pelo comando rustc.